### DISCUSSION TITLE
powerlevel10k_rainbow resize and line issues

### DISCUSSION THREAD

**Discussion by matthijs110** [JanDeDobbeleer/oh-my-posh#3387] [Troubleshoot] [answered]
Hi,

I am experiencing some issues with the [powerlevel10k_rainbow](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/powerlevel10k_rainbow.omp.json) theme within Windows Terminal (please see the GIF):

1. Resizing the terminal results in weird overlapping of the segments. Is this normal behaviour?
2. It might be hard to see, but on both sides the line is interupted. I have seen plenty of examples where this is not the case. How can I fix this? This particular issue does not appear in Visual Studio Code.

**The resizing issue:**
![issue](https://user-images.githubusercontent.com/4161158/213921921-d63cf4ec-2eeb-4915-aa14-e116ce2f6bb2.gif)

**Higher quality image:**
![image](https://user-images.githubusercontent.com/4161158/213922065-b3d1daaf-0d8a-4417-9ee6-01c21119a9d0.png)

I did [tweak](https://gist.github.com/matthijs110/506c74aea48fc34b0c729a2bd46cadd5) the theme, but nothing around those lines at both ends and I am using PowerShell 7.3.1. within Windows Terminal with `CaskaydiaCove Nerd Font Mono` as font family with size 12.

Can somebody help me with one or both issues? Thanks!

**Comment by JanDeDobbeleer** [comment, marked as answer]
Resizing will always make a wonky prompt as the prompt isn't repainted when that happens.

The bars not being connected is a font or font setting problem (size etc).

**Reply by matthijs110** [reply]
Check! My second question is about the small gap in the line on the left and right side of the prompt. Its hard to see on the main image, so here is a zoomed version:
![image](https://user-images.githubusercontent.com/4161158/213927032-e539e14c-d753-44ee-a388-bac5e796dd5a.png)

**Reply by JanDeDobbeleer** [reply]
@matthijs110 I rewrote the original one once I saw the issue:

> The bars not being connected is a font or font setting problem (size etc).

**Reply by matthijs110** [reply]
Yes, thats what I thought to. After some trial and error I decided to switch to another font (FiraCode NFM). Thanks
